### PR TITLE
Format contact.get API response, to show optionValue label instead of value

### PR DESCRIPTION
**Pull request by monishdeb**

### CODE REVIEW

**Comment by monishdeb** [comment]
@seamuslee001 can you please test this PR which is deployed in jma.st? 

Thanks!

File: CRM/Contact/GetAPIWrapper.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+   * the wrapper contains a method that allows you to alter the parameters of the api request (including the action and the entity)
+   */
+  public function fromApiInput($apiRequest) {
+    return $apiRequest;
+  }
+  /**
+   * alter the result before returning it to the caller.
+   */
+  public function toApiOutput($apiRequest, $result) {
+    CRM_Core_Error::debug('result', $result);
```
Comment (seamuslee001) [inline CRM/Contact/GetAPIWrapper.php]: @monishdeb This doesn't look like it is doing any replacement

File: CRM/Contact/GetAPIWrapper.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    return $apiRequest;
+  }
+  /**
+   * alter the result before returning it to the caller.
+   */
+  public function toApiOutput($apiRequest, $result) {
+    foreach ((array) $result['values'] as $k => $value) {
+      foreach ($value as $key => $val) {
+        if (strstr($key, 'custom_')) {
+          $result['values'][$k][$key] = explode(', ', CRM_Core_BAO_CustomField::displayValue($val, $key));
```
Comment (monishdeb) [inline CRM/Contact/GetAPIWrapper.php:16]: @seamuslee001 sorry I forgot to push the latest code. Here's the final patch

**Comment by seamuslee001** [comment]
I would propose declining this in favour of https://github.com/JMAConsulting/autismontario_solr_modifications/pull/6 and https://github.com/JMAConsulting/aosearch_app/pull/8

**Comment by seamuslee001** [comment]
closing as the other 2 PRs got merged

**Pull request closed without merging**